src/test/perf_counters.cc

```diff
@@ -371,15 +359,6 @@ TEST(PerfCounters, TestLabeledCountersOnly) {
                     "nick": "",
                     "priority": 0,
                     "units": "none"
-                },
-                "bar": {
-                    "type": 1,
-                    "metric_type": "gauge",
```

Comment:
This means we're getting rid of this counter from schema as well?

Reply:
@idryomov Do you have any thoughts on filtering out values from a labeled counter that haven't been initialized yet from the `counter dump` command? I'm doing this ensure prometheus isn't filled with unitialized data in labeled counter.

Reply:
I do ;)  `counter dump` and `counter schema` should NOT filter anything.  These commands are about dumping raw data.  Seeing a counter name and `0` next to it in the textual dump is _very_ useful.

If the concern is "filling" prometheus, then doing this in the exporter might be an option but even that is debatable.

Reply:
regardless of what we do for `counter dump`, @avanthakkar is right that `counter schema` shouldn't do any filtering. that just gives you metadata about which counters exist

Reply:
> If the concern is "filling" prometheus, then doing this in the exporter might be an option but even that is debatable.

this filtering may be a premature optimization, but i do believe there's a scale where the bandwidth and database savings would be worth it. it's just hard to make decisions about that until we've done some scale testing

Reply:
@cbodley @alimaredia If we decide to omit dumping  of metrics which aren't initializated, meaning we avoid sending a metric to Prometheus when its value is 0 in a counter dump, this action could potentially disrupt the functionality of Grafana panels. Additionally, any alerts that are configured based on that specific metric name might also be negatively impacted